fc/protocol.py

```diff
@@ -103,21 +100,37 @@ def __init__(self, protoFile, indentLevel=0):
         self.outputs = []
         self.plots = []
 
+        # Information from the model interface
+        self.model_interface = []
+
+        # A mapping of namespace names to uris, for namespaces used in the
+        # protocol
+        self.ns_map = {}
+
         # Parse the protocol file and fill in the structures declared above
         start = time.time()
+
         import fc.parsing.CompactSyntaxParser as CSP
+        CSP.DoXmlImports()
```

Comment:
Why is this needed?

Reply:
I don't know! It doesn't work if that's never called. I think we discussed it was something about the way the compact syntax parser was designed at the moment that we were hoping to eventually change?